src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -150,7 +150,7 @@ export class RgwBucketFormComponent extends CdForm implements OnInit, AfterViewC
         ]
       ],
       lock_mode: ['COMPLIANCE'],
-      lock_retention_period_days: [0, [CdValidators.number(false), lockDaysValidator]],
+      lock_retention_period_days: [10, [CdValidators.number(false), lockDaysValidator]],
```

Comment:
I'm not sold about this change, I feel like this is some important configuration that you want the user to be completly in charge of, so I don't see the appeal to set a default value.

Reply:
0 is an invalid value that means form will not allow setting that. 
So we should put some valid default value IMO
Please see this resolved conversation (just repoened) https://github.com/ceph/ceph/pull/55866#discussion_r1508986280

Reply:
I don't mind the change, but I don't understand the need to set a default value here. What is the benefit? 
To me it feels like something that might cause more harm than beneffit, if you mistakenly create the bucket without modifying to your desired lock_retention_period

Reply:
The idea for default values is https://carbondesignsystem.com/components/form/usage/#default-values

> Where possible, add programmatic assistance. Detect and pre-fill inputs to reduce errors and save time.

Reply:
The idea to set default values, as said, is to reduce errors, which I think this will do the opposite, resolving conversation since this is subjective